src/pages/TrueSightV2/hooks/useKyberAIData.tsx

```diff
@@ -57,6 +57,7 @@ const kyberAIApi = createApi({
       },
       providesTags: (result, error, arg) => (arg.watchlist === true ? ['myWatchList', 'tokenList'] : ['tokenList']),
     }),
+    //2.
```

Comment:
what is this bro?

Reply:
just naming for the APIs, there are 25 APIs in this file sometimes hard to know which one is